docker-entrypoint.sh

```diff
@@ -1,9 +1,45 @@
 #!/bin/bash
 set -e
 
-if [ ! -e '/var/www/html/version.php' ]; then
-    tar cf - --one-file-system -C /usr/src/nextcloud . | tar xf -
+# version_greater A B returns whether A > B
+function version_greater() {
+	[[ "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1" ]];
+}
+
+installed_version="0.0.0"
```

Comment:
No it won't -- I work with rpm-style versions quite a lot. `sort -V` does version checks against the leading digits first. In general you would use `~` for things like `1.0.0~rc1` which is a **lower** version number than `1.0.0` (or `1.0.0~rc2`). However something like `0.0.0+rc1` is a **higher** version number than `1.0.0`.